types.d.ts

```diff
@@ -944,25 +944,34 @@ export interface LoginUrl {
   request_write_access?: Boolean;
 }
 
+export namespace CallbackQuery {
+  interface AbstractCallbackQuery {
+    /** Unique identifier for this query */
+    id: String;
+    /** Sender */
+    from: User;
+    /** Message with the callback button that originated the query. Note that message content and message date will not be available if the message is too old */
+    message?: Message;
+    /** Identifier of the message sent via the bot in inline mode, that originated the query. */
+    inline_message_id?: String;
```

Comment:
Left un-unionized, intentionally? Feel free to force push if not

Reply:
See my previous comment: https://github.com/KnorpelSenf/typegram/pull/6#issuecomment-680128179